test/eq.test.js

```diff
@@ -1,21 +1,135 @@
-import assert from 'assert';
-import eq from '../eq.js';
-
-describe('eq', function() {
-  it('should perform a `SameValueZero` comparison of two values', function() {
-    assert.strictEqual(eq(), true);
-    assert.strictEqual(eq(undefined), true);
-    assert.strictEqual(eq(0, -0), true);
-    assert.strictEqual(eq(NaN, NaN), true);
-    assert.strictEqual(eq(1, 1), true);
-
-    assert.strictEqual(eq(null, undefined), false);
-    assert.strictEqual(eq(1, Object(1)), false);
-    assert.strictEqual(eq(1, '1'), false);
-    assert.strictEqual(eq(1, '1'), false);
-
-    var object = { 'a': 1 };
-    assert.strictEqual(eq(object, object), true);
-    assert.strictEqual(eq(object, { 'a': 1 }), false);
-  });
-});
+import assert from 'assert'
+import eq from '../eq.js'
+
+describe('eq', () => {
+  it('should return true without arguments', () => {
+    assert.strictEqual(eq(), true)
+  })
+
+  it('should treat missing argument as undefined', () => {
+    assert.strictEqual(eq(undefined), true)
+  })
+
+  describe('when types are different', () => {
+    it('should return false to number and boolean', () => {
+      assert.strictEqual(eq(0, false), false)
+    })
+
+    it('should return false to null and undefined', () => {
+      assert.strictEqual(eq(null, undefined), false)
+    })
+
+    it('should return false to number and object', () => {
+      assert.strictEqual(eq(1, Object(1)), false)
+    })
+
+    it('should return false to number and string', () => {
+      assert.strictEqual(eq(1, '1'), false)
+    })
+
+    it('should return false to string and symbol', () => {
+      assert.strictEqual(eq(Symbol('abc'), 'abc'), false)
+    })
+  })
+
+
+  describe('when type is undefined', () => {
+    it('should return true for both undefined', () => {
+      assert.strictEqual(eq(undefined, undefined), true)
```

Comment:
Unecessary test.

This is the same test defined at line 10. Put undefined in the second arg or not gives the same result

Reply:
Indeed. I tried to keep the tests that were implemented before.  One of them is the one with only one argument that you refer to. If you recall, the first test is also the same as these two you talk about, as passing no arguments at all is also equivalent to `undefined, undefined`.

I actually do not know to which extent those tests with missing arguments make sense. How about removing both of them?

Reply:
That is ok for me, we can remove both